plugins/janus_videoroom.c

```diff
@@ -1152,12 +1163,16 @@ json_t *janus_videoroom_query_session(janus_plugin_session *handle) {
 				json_object_set_new(media, "data", json_integer(participant->data));
 				json_object_set_new(info, "media", media);
 				json_object_set_new(info, "bitrate", json_integer(participant->bitrate));
-				if(participant->arc || participant->vrc || participant->drc) {
+				if(participant->arc || participant->vrc[0] || participant->drc) {
 					json_t *recording = json_object();
 					if(participant->arc && participant->arc->filename)
 						json_object_set_new(recording, "audio", json_string(participant->arc->filename));
-					if(participant->vrc && participant->vrc->filename)
-						json_object_set_new(recording, "video", json_string(participant->vrc->filename));
+					if(participant->vrc[0] && participant->vrc[0]->filename)
+						json_object_set_new(recording, "video", json_string(participant->vrc[0]->filename));
+					if(participant->vrc[1] && participant->vrc[1]->filename)
+						json_object_set_new(recording, "video-sim1", json_string(participant->vrc[1]->filename));
+					if(participant->vrc[2] && participant->vrc[0]->filename)
```

Comment:
Yep, definitely a typo, thx for spotting this!

As a side note, we'll really have to decide what the best way to handle recordings and RTP forwarding will be, with Simulcasting enabled. Both are not an issue when we receive a single stream, but can be with three different substreams that you're not sure will all arrive. At the moment I'm simply forwarding the lowest substream when RTP forwarding (as it's the only one we know for sure is always sent) while for recording I'm saving each substream to a different file. That's definitely not my favourite approach, as while it's very useful for debugging (I can check what was sent for each of them), it much more problematic to handle when postprocessing (how do you "merge" them to be sure you always have the "best quality" at any moment?).

Probably I'll have to treat both as a "user with no trouble at all", and find a way to automatically drop or go up substreams as I notice them appearing/going away.